mrn_mysql_compat.h

```diff
@@ -982,12 +982,21 @@ typedef uint mrn_srid;
   (table_list->get_db_name())
 #endif
 
-#if defined(MRN_MARIADB_P) &&                                   \
+#if defined(MRN_MARIADB_P) &&                                        \
     (MYSQL_VERSION_ID >= 110400 && MYSQL_VERSION_ID < 110500)
   using mrn_io_and_cpu_cost = IO_AND_CPU_COST;
 #  define MRN_HANDLER_HAVE_MULTI_RANGE_READ_INFO_CONST_LIMIT
 #  define MRN_HANDLER_HAVE_KEYREAD_TIME
+#  define MRN_WARN_DEPRECATED(thd, what, to)                         \
+   (warn_deprecated<999999>(thd, what, to))
```

Comment:
Could you add a comment why we always use `999999`?

Reply:
I see.